gapic-generator/Rakefile

```diff
@@ -25,9 +26,41 @@ Rake::TestTask.new :test do |t|
   t.test_files = FileList["test/**/*_test.rb"]
 end
 
+desc "Compile the necessary protobuf files."
+task :compile_protos do
```

Comment:
Unfortunately I had to roll my own protoc task since the ruby-protobuf blindly globbed all .proto files and the submoduled repos contain a lot more protos than we need.